tests/by-util/test_mv.rs

```diff
@@ -1180,6 +1180,33 @@ fn test_mv_overwrite_nonempty_dir() {
     assert!(at.dir_exists(dir_b));
 }
 
+#[test]
+fn test_mv_nonempty_directory_exists() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    let dir_a = "test_mv_nonempty_directory_exists_a";
+    let dir_b = "test_mv_nonempty_directory_exists_b";
+    let dummy = "test_mv_nonempty_directory_exists_a/file";
+    let dummy2 = "test_mv_nonempty_directory_exists_b/file";
+    at.mkdir(dir_a);
+    at.mkdir(dir_b);
+    at.touch(dummy);
+    at.touch(dummy2);
+
+    // Not same error as GNU; the error message is custom.
+    // GNU: "mv: Couldn't move (Directory not empty; from=a; to=b)
+    // Current: "mv: Couldn't move {} to {}: A directory exists with the same name at destination"
+
+    // Verbose output for the move should not be shown on failure
+    let result = ucmd.arg("-vT").arg(dir_a).arg(dir_b).fails();
```

Comment:
could you please also verify the error message ?

Reply:
Ok now I see there is an issue with the verbose flag, it reverts to the other message. fixing

Reply:
sorry, I meant that you should verify that the output (stderr probably) contains  "cannot overwrite" too :)

Reply:
Yeah I understood, but then II noticed passing the `-T` flag will cause it to revert to the other error message...
But running the same test without the `-T` (or `-vT` ) flag, it actually says that the assertion fails that the cmd even failed
 ```Rust
    ucmd.arg(dir_a)
        .arg(dir_b)
        .fails() // <-here
```
despite when running the command `cargo run -- mv test1 ..` (which is setup to trigger this behavior, the same way the tests are setup), it will produce the correct message and indeed fail properly. 

So instead of mapping the error context as before on that branch, I tried just returning
```Rust
USimpleError::new(1, format!("cannot overwrite {}:{}", targetpath.quote(), e))
```
which will also return the correct error message when running the command, but still fails the test on `fails()` assertion... obviously I'm just missing something, I'll figure it out.